index.bs

```diff
@@ -989,24 +876,10 @@ list=]</dfn> |list|, run these steps:
      not `"*"`: Return `false`.
   1. Let |element| be the |attribute|'s {{Element}}.
   1. Let |element name| be |element|'s [=Element/local name=].
-  1. If |element| is a in either the [=SVG namespace|SVG=] or
```

Comment:
I am wondering if we should explicitly `return false` for any _attribute_ that is not in the null namespace for now. (Similar to how we check for the HTML namespace of elements)

Reply:
Oh, I guess HTML elements never have namespaced attributes anyway.

Reply:
Yeah, that'd make sense.

(OTOH, I don't expect this wording to remain in the spec for long. I just want to remove the old text first.)

Reply:
Added a line that checks for attribute namespaces.